src/components/Sidebar/Sidebar.story.tsx

```diff
@@ -14,44 +20,66 @@ export default {
   component: Sidebar,
   argTypes: {
     state: { control: { disable: true } },
+    labelAsTooltip: { control: 'boolean' },
   },
 };
 
 const Header = styled(H)`
   margin-top: 0;
 `;
 
-const Template: Story<Props> = ({ attach = 'left', ...args }) => {
+const Template: Story<
+  Props & Pick<ItemPropsIntrinsic, 'labelAsTooltip'>
+> = ({ attach = 'left', labelAsTooltip, ...args }) => {
   const style =
     attach === 'right'
       ? ({ position: 'absolute', top: 0, right: 0 } as const)
       : null;
-
   return (
     <Layout.FullBleed>
       <Sidebar attach={attach} {...args} style={style}>
-        <Sidebar.Item label="Item 1" icon={<Gear />}>
-          <Header size="3">Item 1</Header>
+        <Sidebar.Item
+          label="Privacy"
+          labelAsTooltip={labelAsTooltip}
```

Comment:
One more thing: you should use the `labelAsTooltip` on the `args` object so the storybook control can work... so instead of just `labelAsTooltip` just change this to `args.labelAsTooltip`